cpp/models/abm/testing_strategy.h

```diff
@@ -94,35 +84,14 @@ class TestingCriteria
     /**
      * @brief Check if a Person and a Location meet all the required properties to get tested.
      * @param[in] p Person to be checked.
-     * @param[in] l Location to be checked.
      * @param[in] t TimePoint when to evaluate the TestingCriteria.
      */
-    bool evaluate(const Person& p, const Location& l, TimePoint t) const;
+    bool evaluate(const Person& p, TimePoint t) const;
 
 private:
-    /**
-     * @brief Check if a Person has the required age to get tested.
-     * @param[in] p Person to be checked.
-     */
-    bool has_requested_age(const Person& p) const;
-
-    /**
-     * @brief Check if a Location is in the set of Location%s that are allowed for testing.
-     * @param[in] l Location to be checked.
-     */
-    bool is_requested_location_type(const Location& l) const;
-
-    /**
-     * @brief Check if a Person has the required InfectionState to get tested.
-     * @param[in] p Person to be checked.
-     * @param[in] t TimePoint when to check.
-     */
-    bool has_requested_infection_state(const Person& p, TimePoint t) const;
-
-    std::vector<AgeGroup> m_ages; ///< Set of #AgeGroup%s that are either allowed or required to be tested.
-    std::vector<LocationType> m_location_types; /**< Set of #LocationState%s that are either allowed or required to be 
-    tested.*/
-    std::vector<InfectionState> m_infection_states; /**< Set of #InfectionState%s that are either allowed or required to
+    std::unordered_set<size_t> m_ages; ///< Set of #AgeGroup%s that are either allowed or required to be tested.
```

Comment:
Bitset was suggested by Daniel to avoid using vectors for both AgeGroup and InfectionState.
However, now the number of AgeGroup is not defined until runtime so bitset is not possible.